default/scripting/buildings/shipyards/ORBITAL_DRYDOCK.focs.txt

```diff
@@ -35,6 +35,7 @@ BuildingType
                 Source
                 ContainedBy And [
                     Planet
+                    [[PLANET_OWNED_DRYDOCK_HIGHEST_HAPPINESS]]
```

Comment:
Correct me if I'm wrong, but isn't the `Planet` condition redundant here, as `PLANET_OWNED_DRYDOCK_HIGHEST_HAPPINESS` already matches only planets...?